src/index.js

```diff
@@ -8,7 +8,7 @@ const NEAR_NET_VERSION_TEST = '98';
 const utils = require('./utils');
 const nearToEth = require('./near_to_eth_objects');
 
-const GAS_AMOUNT = new BN('1000000000000000000');
+const GAS_AMOUNT = new BN('1000000000000000'); // 10e14.
```

Comment:
nearlib has 10e17 as default gas and works, this is unlikely causing the gas usage problem by itself

Reply:
nearcore has 1e15 as max gas (https://github.com/nearprotocol/nearcore/blob/master/neard/res/genesis_config.json#L16) so don't know how nearlib works.